freetextresponse/private/edit.less

```diff
@@ -1,3 +1,3 @@
-.xblockfreetext_edit {
+.freetextresponse_edit {
```

Comment:
If we aren't using this file, we can remove it here, as well as any references elsewhere.